lib/get-editor-config.js

```diff
@@ -16,8 +16,8 @@ class EditorConfigResolver {
      * comment: ;this is a comment
      */
     this.regex = {
-      section: /^\s*\[(([^#;]|\\#|\\;)+)\]\s*([#;].*)?$/,
-      param: /^\s*([\w.\-_]+)\s*[=:]\s*(.*?)\s*([#;].*)?$/,
+      section: /^\s*\[(([^#;]|\\#|\\;)+)]\s*([#;].*)?$/,
+      param: /^\s*([\w.-]+)\s*[:=]\s*(.*?)\s*([#;].*)?$/,
```

Comment:
A _ seems to disappear here, is it okay?

Reply:
Looks like it's because `\w` already includes `_`:

> The \w metacharacter is used to find a word character. A word character is a character from a-z, A-Z, 0-9, including the _ (underscore) character.

https://www.w3schools.com/jsref/jsref_regexp_wordchar.asp